sklearn/preprocessing/imputation.py

```diff
@@ -112,9 +112,14 @@ class Imputer(BaseEstimator, TransformerMixin):
     ----------
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature if axis == 0.
+        The values corresponding to features ``all_missing_` is ``nan``.
+
+    all_missing_ : array of shape (n_features - n_features_new, )
+        The features with all missing values for axis = 0
 
     imputed_features_ : array of shape (n_features_with_missing, )
```

Comment:
I'm not sure this is a good name for it. Maybe `indicated_features_` or `incomplete_features_`. one problem is that nothing is imputed during fit.